Mixpanel/MixpanelInstance.swift

```diff
@@ -197,15 +197,20 @@ open class MixpanelInstance: CustomDebugStringConvertible, FlushDelegate {
     var timedEvents = InternalProperties()
     var serialQueue: DispatchQueue!
     var taskId = UIBackgroundTaskInvalid
-    let flushInstance = Flush()
+    let flushInstance: Flush
     let trackInstance: Track
-    let decideInstance = Decide()
+    let decideInstance: Decide
 
-    init(apiToken: String?, launchOptions: [UIApplicationLaunchOptionsKey : Any]?, flushInterval: Double) {
+    init(apiToken: String?, launchOptions: [UIApplicationLaunchOptionsKey : Any]?, flushInterval: Double, name: String=UUID().uuidString) {
```

Comment:
we don't need a default value here for name because it's already getting one in MixpanelManager